README.md

```diff
@@ -16,10 +16,40 @@ npm install
 npm start
 ```
 
-- Run debug start:
+- Run debug start (Attach mode):
 
 ```bash
 npm run start:debug
 ```
 
-- And run .vscode/launch.json
+- And run .vscode/launch.json (Debug Attach)
+
+# Debugging state
+
+Since VS Code v1.47, it enables a new debugging feature, [New JavaScript Debugger](https://code.visualstudio.com/updates/v1_47#_debugging) as default.
+
+This new debugger works with no configuration on this way:
+
+![create-js-debug-terminal](./readme-resources/00-create-js-debug-terminal.png)
+
+And run command:
+
+```bash
+npm start
+```
+
+- It runs the command with auto attached debugger
+
+![running-debugger](./readme-resources/01-running-debugger.png)
+
+- But it stops on transpiled code break points when we are debugging:
+
+![stop-on-transpiled-code](./readme-resources/02-stop-on-transpiled-code.png)
+
```

Comment:
This is something that has been fixed in a nightly build and likely to be released in the near future, [more information](https://github.com/microsoft/vscode/issues/103048)